dist/leaflet.css

```diff
@@ -202,6 +202,16 @@
 .leaflet-interactive {
 	cursor: pointer;
 	}
+
+.leaflet-zoomPixelated

```

Comment:
To keep with naming standard in our CSS, this should be `leaflet-zoom-pixelated`.